client/src/components/ProjectUsers.vue

```diff
@@ -0,0 +1,286 @@
+<script>
+import { mapState, mapActions, mapMutations } from 'vuex';
+import UserAvatar from '@/components/UserAvatar.vue';
+import djangoRest from '@/django';
+
+export default {
+  name: 'ProjectUsers',
+  components: {
+    UserAvatar,
+  },
+  inject: ['user'],
+  data: () => ({
+    showAddMemberOverlay: false,
+    showAddCollaboratorOverlay: false,
+    selectedPermissionSet: {},
+  }),
+  computed: {
+    ...mapState(['currentProject', 'allUsers']),
+    permissions() {
+      return this.currentProject.settings.permissions;
+    },
+    writePermission() {
```

Comment:
This is only referenced twice, so I think it might be cleaner to just use `user.is_superuser` instead.